src/modules/reservation.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable operator-linebreak */
+/* eslint-disable no-use-before-define */
+/* eslint-disable camelcase */
```

Comment:
- [ ] Please remove the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :+1: